src/Symfony/Bundle/FrameworkBundle/Resources/config/psr-http-message.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+
+        <service id="argument_resolver.psr_message" class="Symfony\Component\HttpKernel\Controller\ArgumentResolver\PsrServerRequestArgumentResolver">
```

Comment:
I'm also not sure if the naming is chosen really well. Imagine that someone actually has packages under an `ArgumentResolver` vendor. Their service names will then conflict with these.